src/repository.c

```diff
@@ -1237,6 +1263,20 @@ int git_repository_set_index(git_repository *repo, git_index *index)
 	return 0;
 }
 
+int git_repository_grafts__weakptr(git_grafts **out, git_repository *repo)
+{
+	assert(out && repo && repo->shallow_grafts);
```

Comment:
Cut & paste error here?
Presumably this should be
`assert(out && repo && repo->grafts);`